src/System.Linq.Queryable/src/System/Linq/Queryable.cs

```diff
@@ -1609,5 +1609,35 @@ public static TSource Aggregate<TSource>(this IQueryable<TSource> source, Expres
                     null,
                     CachedReflectionInfo.Aggregate_TSource_TAccumulate_TResult_4(typeof(TSource), typeof(TAccumulate), typeof(TResult)), source.Expression, Expression.Constant(seed), Expression.Quote(func), Expression.Quote(selector)));
         }
+
+        public static IQueryable<TSource> SkipLast<TSource>(this IQueryable<TSource> source, int count)
+        {
+            if (source == null)
+            {
```

Comment:
Nit: For consistency in this file, maybe get rid of `{` and `}` and the additional whitespace below?